README.md

```diff
@@ -1,78 +1,38 @@
 # DevTools RFCs
 
-Most changes to our tools, including bug fixes and documentation improvements, can be implemented and reviewed via the normal GitHub pull request workflow.
+Some changes to DevTools need feedback from the team and community to reach a consensus. In such a case you should use this repository to discuss the cahnge proposal proposal.
 
-Some changes though are "substantial", and we ask that these be put through a bit of a design process and produce a consensus among the [DevTools core team][].
+DevTools RFC process aims to be lightweight and focused on conversations. Therefore opening a DevTools RFC is as easy as opening an issue in this repository to discuss the proposal.
 
-The "RFC" (request for comments) process is intended to provide a consistent and controlled path for new features to enter the framework.
+The [DevTools core team][] will be ultimately responsible for accepting or rejecting a change proposal, but the discussion is open to everybody.
 
-[Active RFC List][]
+The list of RFCs currently under discussion can be found at: [Active RFC List][].
 
-## When you need to follow this process
+## Creating a RFC
 
-You need to follow this process if you intend to make "substantial" changes to DevTools or its documentation. What constitutes a "substantial" change is evolving based on community norms, but may include the following.
+* Create an issue with the "proposal" label
+* Get people's attention. Some options:
+    * Send an email to [dev-developer-tools][]
```

Comment:
This should be a link to the info page so people know how to subscribe to it: https://lists.mozilla.org/listinfo/dev-developer-tools